tests/torch/quantization/test_hawq_precision_init.py

```diff
@@ -311,6 +311,11 @@ def __str__(self):
 INCV3_BITS_COMPLEXITY = map(lambda x, y: x * y, INCV3_FLOPS_PER_MODULE, INCV3_BITWIDTH_PER_MODULE)
 INCV3_COMPRESSION_RATIO = sum(INCV3_FLOPS_PER_MODULE) * 8 / sum(INCV3_BITS_COMPLEXITY)
 
+
+def pretrained_inception_v3():
+    return inception_v3(pretrained=True)
```

Comment:
Without `pretrained` option, the initialization of weights is different from what we had with previous version of torchvision.